src/obs-plugin/plugin-main.cpp

```diff
@@ -102,8 +124,25 @@ bool obs_module_load(void)
             action->setText(obs_module_text("Start Virtual Camera"));
             obs_output_stop(output);
         } else {
-            action->setText(obs_module_text("Stop Virtual Camera"));
-            obs_output_start(output);
+            if(isSupportedResolution()){
+                action->setText(obs_module_text("Stop Virtual Camera"));
+                obs_output_start(output);
+            } else {
+                
+                stringstream msg;
+                msg << obs_module_text("Your output resolution not supported. Please use one of the following:") << endl;
+                for(string res : knownResolutions){
+                    msg << res << endl;
+                }
+                //QString title = QString::fromStdString(obs_module_text("UnsupportedResolution_Title"));
```

Comment:
Remove for now (let's handle localizations separately)

Reply:
oof forgot that one